spec/models/user_spec.rb

```diff
@@ -1246,6 +1246,112 @@ def create_user(options = {})
 
   end
 
+  describe '#erase' do
+    subject { user.erase }
+
+    let(:user) { FactoryBot.build(:user) }
+
+    context 'the update is successful' do
+      before do
+        user.close!
```

Comment:
Should we create a `closed` factory trait?

Reply:
Ah, already had one! Updated.